tests/test_splitter.py

```diff
@@ -8,8 +8,57 @@
 
 class TestSplitter(TestCase):
 
-    def test_Split(self):
-        sp.Splitter.Split("filepath")
+    def test_Split_one(self):
```

Comment:
@ASAAR one test is for a file that is "," delimited, one is for a file that is ";" delimited. if you'd like I can put both test cases under one function, and have it called as before, or I can just specify in the function names which is which. Let me know what is easier for you.